packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1656,9 +1656,7 @@ idx.error.code.access_denied.device_assurance.remediation.android.zero.trust.and
 
 # Device Assurance Grace Period
 idx.device_assurance.grace_period.title = Device assurance reminder
-# {0} is the number of days until the grace period expires
-idx.device_assurance.grace_period.warning.title.due_by_days = Your device doesn't meet the security requirements. Fix the issue within {0} days to prevent lockout.
```

Comment:
We are consolidating to a single string for the expiration message. This can get deleted in the next translation run